torchvision/prototype/datasets/_builtin/caltech.py

```diff
@@ -95,8 +96,8 @@ def _collate_and_decode_sample(
         image = decoder(image_buffer) if decoder else image_buffer
 
         ann = read_mat(ann_buffer)
-        bbox = torch.as_tensor(ann["box_coord"].astype(np.int64))
-        contour = torch.as_tensor(ann["obj_contour"])
+        bbox = BoundingBox(ann["box_coord"].astype(np.int64).squeeze()[[2, 0, 3, 1]], format="xyxy")
```

Comment:
Let's not block this here, but I was wondering what we set as `image_size` if the user specifies `decoder=None`? If not set, the `BoundingBox` will default to an `image_size` that is barely able to contain it.

Reply:
It is possible to get the image sizes without fully decoding the image. We haven't implemented it in torchvision yet but I think we should. So we could always return the correct image properties here